sublime_linter.py

```diff
@@ -243,6 +243,7 @@ def highlight(self, view, errors, hit_time):
             return
 
         bid = view.buffer_id()
+        errors.sort(key=lambda x: (x["line"], x["start"], x["end"]))
         persist.raw_errors[bid] = errors
```

Comment:
With this, you should mention somewhere (in the docstring/comment of `persist.raw_errors` most likely) that its values are supposed to be sorted at all times (an invariant).

Reply:
I already mentioned that we don't sort here. The order of the errors is arbitrary as results fly in. Also this particular implementation mutates errors.

Reply:
OK. This is why I did not write a docstring. Moving it back and using `sorted( `.

Reply:
But sorting was added in https://github.com/SublimeLinter/SublimeLinter/pull/823/commits/c8f5472c233e286862b597b5697bf95bb06bd0a1, unless I'm mistaken?